src/desktop/drbd.desktop

```diff
@@ -14,6 +14,7 @@ X-SuSE-YaST-AutoInst=all
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=drbd
+X-SuSE-YaST-Keywords=high,availability,ha
```

Comment:
`high` is probably too generic, I'd add `storage`, `block`, `device`, `cluster`.